plugins/modules/azure_rm_webapp.py

```diff
@@ -234,6 +235,16 @@
           registry_server_user: user
           registry_server_password: pass
 
+    - name: Create a multi-container web app
+      azure_rm_webapp:
+        resource_group: myResourceGroup
+        name: myMultiContainerWebapp
+        plan: myAppServicePlan
+        app_settings:
+          testkey: testvalue
+        container_settings:
+          name: "COMPOSE|{{ lookup('file', 'docker-compose.yml') | b64encode }}"
```

Comment:
COMPOSE and DOCKER should be followed by lowercase。
```suggestion
          name: COMPOSE{{ lookup('file', 'docker-compose.yml') | b64decode }}
```

Reply:
To my understanding, Azure webapp API uses base64 encoded docker-compose file content with `COMPOSE|` prefix as the container name when deploying multi-container application. For example: https://docs.microsoft.com/en-us/azure/app-service/tutorial-multi-container-app#update-app-with-new-configuration